Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Pages/Documentation.razor

```diff
@@ -84,8 +84,8 @@ JSON Lambda event document as the POST request's body.
 </p>
 
 <pre style="white-space: pre-wrap; background-color:lightgray;font-size:85%">
-&gt; dotnet lambda-test-tool-3.1 --help
-AWS .NET Core 3.1 Mock Lambda Test Tool (0.10.0)
+&gt; dotnet lambda-test-tool-6.0 --help
+AWS .NET Core 6.0 Mock Lambda Test Tool (0.14.0)
```

Comment:
This could bring a little confusion regarding the runtime version. While the project may appear to be running on .NET 6, when actually can be configured for version 8 ^_^